api/sqlc.yaml

```
@@ -0,0 +1,15 @@
+    queries: "queries/"
+    schema: "migrations/"
+    gen:
+      go:
+        package: "database"
+        out: "infrastructure/database"
+        sql_package: "pgx/v5"
+        emit_json_tags: true
+        emit_prepared_queries: false
+        emit_interface: true
+        emit_exact_table_names: false
+        emit_empty_slices: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)